google-signin.html

```diff
@@ -118,73 +118,73 @@
       WIDE: 'wide'
     };
 
-/**
-&lt;google-signin&gt; is used to authenticate with Google, allowing you to interact
-with other Google APIs such as Drive and Google+.
-
-<img style="max-width:100%;" src="https://cloud.githubusercontent.com/assets/107076/6791176/5c868822-d16a-11e4-918c-ec9b84a2db45.png"/>
-
-If you do not need to show the button, use companion `<google-signin-aware>` element to declare scopes, check authentication state.
+    /**
+    `<google-signin>` is used to authenticate with Google, allowing you to interact
+    with other Google APIs such as Drive and Google+.
 
-#### Examples
+    <img style="max-width:100%;" src="https://cloud.githubusercontent.com/assets/107076/6791176/5c868822-d16a-11e4-918c-ec9b84a2db45.png"/>
 
-    <google-signin client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
+    #### Examples
 
-    <google-signin label-signin="Sign-in" client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
+        <google-signin client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
 
-    <google-signin theme="dark" width="iconOnly" client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
+        <google-signin label-signin="Sign-in" client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
 
+        <google-signin theme="dark" width="iconOnly" client-id="..." scopes="https://www.googleapis.com/auth/drive"></google-signin>
 
-#### Notes
+    #### Notes
```

Comment:
It's probably easier if we just make the docs a html comment like https://github.com/GoogleWebComponents/google-sheets/blob/master/google-sheets.html#L7.